homeassistant/components/foscam/switch.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         """Initialize a Foscam Sleep Switch."""
         super().__init__(coordinator, config_entry.entry_id)
 
-        self._attr_unique_id = "sleep_switch"
+        self._attr_unique_id = f"{DOMAIN}_{config_entry.entry_id}_sleep_switch"
```

Comment:
You don't need to add the domain to it, since unique ids are unique per domain per platform